### PR TITLE
Clear class that no use in the UI library

### PR DESCRIPTION
**Pull request by IgnacioRodrigues**
In this pull request, I worked to eliminate classes that are no longer useful, this is because I am working on cleaning the UI library so that the code is as clean and neat as possible. In this case, only the call to the class "link - title" was deleted

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
         <p className="content-subtitle">{_('login.enter_doppler_sub')}</p>
         <p className="content-subtitle">
           {_('login.you_want_create_account')}{' '}
-          <Link to="/signup" className="link--title">
+          <Link to="/signup">
```
Comment (andresmoschini) [inline src/components/Login/Login.js]: Me parece que esto no es un breaking change.

Comment (andresmoschini) [reply src/components/Login/Login.js]: Menos para la webapp, donde el cliente es un usuario humano (en general más tolerante a los cambios que una máquina)

Comment (IgnacioRodrigues) [reply src/components/Login/Login.js]: Que le pondrías vos? Que es un refactor?

Comment (cbernat) [reply src/components/Login/Login.js]: If we are eliminanting redundant classes we are not changing nothing substantial, I woudl say this is a chore.

**Comment by andresmoschini** [comment]
👍

**Comment by andresmoschini** [comment]
:shipit: 

But, you will need to prettify it before:

image

**Comment by IgnacioRodrigues** [comment]
> But, you will need to prettify it before:

Yes! Now run prettier 🐱‍💻

**Pull request merged**